tools/importer-rest-api-specs/components/dataapigeneratorjson/models/terraform_resource_definition.go

```diff
@@ -0,0 +1,56 @@
+package models
+
+// ResourceDefinition describes a Resource with information specific to Terraform
+type ResourceDefinition struct {
+	// DisplayName specifies the human-readable name for this Resource, used in the Documentation. (e.g. Load Test)
+	DisplayName string `json:"DisplayName"`
+
+	// ResourceIdName specifies the name of the Resource ID type used for this Resource.
+	ResourceIdName string `json:"Id"`
```

Comment:
This one should match the field name:

```suggestion
	ResourceIdName string `json:"resourceIdName"`
```